packages/genesys-spark-components/src/components/stable/gux-card/tests/gux-card.e2e.playwright.ts

```diff
@@ -0,0 +1,6 @@
+import { checkRenders, test } from '@test/playwrightTestUtils';
+import { renderConfigs } from './gux-card.common';
+
+test.describe('gux-card', () => {
+  checkRenders(renderConfigs, 'gux-card');
```

Comment:
I think we might want `checkRenders(renderConfigs);` here as the current screenshots are cutting off the raised example too much.

Reply:
Will throw up a PR for this 👍